base/docs/utils.jl

```diff
@@ -270,12 +282,17 @@ const builtins = ["abstract", "baremodule", "begin", "bitstype", "break",
 
 moduleusings(mod) = ccall(:jl_module_usings, Any, (Any,), mod)
 
-filtervalid(names) = filter(x->!ismatch(r"#", x), map(string, names))
+isvalid(name::AbstractString) = !ismatch(r"#", name)
```

Comment:
there's already a function named `isvalid`, probably not a good idea to shadow it here